pod.go

```diff
@@ -339,8 +339,6 @@ type Pod struct {
 	runPath    string
 	configPath string
 
-	url string
-
```

Comment:
> The only case would be the proxy opening different sockets for different processes. Every time we call Register or Attach to reach the proxy, we receive a URL, we can assume it could be different for some reasons (maybe performance).

As processes from a given pod must be running in the same pod, they all should be handled through the same proxy. There will be one socket per proxy, and thus one URL per pod.

Without digging into the code, `pod.url` should be set by `hyperstart.go` when we start the agent, i.e. when we start a pod. That should be part of a pod state, and we should be storing that anyway.